tests/reporting/test_universe.py

```diff
@@ -10,13 +11,139 @@ def test_universe_creation(contractsFixture):
     assert universe.getForkEndTime() == 0
     assert reputationToken.getUniverse() == universe.address
     assert reputationToken.getTopMigrationDestination() == longToHexString(0)
-
-def test_universe_name(contractsFixture):
-    universe = contractsFixture.createUniverse(3, "5")
     assert universe.getTypeName() == stringToBytes('Universe')
+    assert universe.getForkEndTime() == 0
+    assert universe.getChildUniverse("5") == longToHexString(0)
+
+def test_get_reporting_window(contractsFixture):
+    universe = contractsFixture.createUniverse(1, "10")
```

Comment:
Consider using a snapshot/fixture for this instead of creating a new universe for each test.  Check out the new fixture stuff I added, or ask me about it in #dev for details.  This will let you have tests that run notably faster because they aren't spinning up the entire world every time and instead spin it all up once and then take a snapshot, then reset for each test.  Resetting a snapshot is very cheap compared to actually doing on-chain work.

Reply:
Alright, I’ll make the change to add a new fixture to increase test run speed.